server/polar/exceptions.py

```diff
@@ -3,29 +3,33 @@
 IntegrityError = sqlalchemy.IntegrityError
 
 
-class AuthenticationRequired(Exception):
```

Comment:
I took the liberty to remove the ones that were unused: `AuthenticationRequired`, `InvalidPlatform`, `InvalidRequest` and `DatabaseRecordExists`.